base/show.jl

```diff
@@ -585,7 +585,7 @@ function show(io::IO, tn::Core.TypeName)
 end
 
 show(io::IO, ::Nothing) = print(io, "nothing")
-print(io::IO, ::Nothing) = throw(ArgumentError("`nothing` should not be printed; use `show`, `repr`, or custom output instead."))
+print(io::IO, ::Nothing) = print(io, "nothing")
```

Comment:
This can be deleted since print falls back to show.

Reply:
```suggestion
```